store/roles/actions.js

```diff
@@ -17,6 +18,12 @@ export default {
     setPrivileges({ commit }, value) {
         commit(types.SET_ROLE_PRIVILEGES, value);
     },
+    getRoles({ commit, rootState }, params) {
+        const { language: userLanguageCode } = rootState.authentication.user;
+        return this.app.$axios.$get(`${userLanguageCode}/roles`, { params }).then(({ collection: roles }) => {
+            commit(types.SET_ROLES, arrayToObject(roles, 'id', 'name'));
```

Comment:
roles: [] as it is in state => Keep the constant - btw. why would u need Object ?